samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,146 @@
+# {{ cookiecutter.project_name }}
+
+This is a sample template for {{ cookiecutter.project_name }}
+
+
+## Requirements
+
+* AWS CLI already configured with Administrator permission
+* [Docker installed](https://www.docker.com/community-edition)
+* [SAM CLI installed](https://github.com/awslabs/aws-sam-cli)
+* [.NET Core installed](https://www.microsoft.com/net/download)
+* [PowerShell Core installed](https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell?view=powershell-6)
+* [AWSLambdaPSCore PowerShell module](https://www.powershellgallery.com/packages/AWSLambdaPSCore/1.1.0.0)
+* [Pester PowerShell module](https://github.com/pester/Pester)
```

Comment:
Is Pester the most used test framework? We need to start customers off on the right path.